From: Jakob Schiotz <schiotz@fysik.dtu.dk>
Date: Wed Sep  3 11:07:13 CEST 2025

Remove one variant of a test that fails due to external compilation failing.

--- tests/cupy_tests/core_tests/test_raw.py.orig	2025-09-02 15:01:15
+++ tests/cupy_tests/core_tests/test_raw.py	2025-09-02 15:02:08
@@ -1109,8 +1109,6 @@
     {'backend': 'nvrtc', 'in_memory': True},
     # this run will force recompilation
     {'backend': 'nvrtc', 'in_memory': True, 'clean_up': True},
-    # Finally, we test NVCC
-    {'backend': 'nvcc', 'in_memory': False},
 )
 class TestRaw(_TestRawBase, unittest.TestCase):
     pass
